microsoft-edge/webview2/concepts/frames.md

```diff
@@ -300,11 +300,13 @@ See also:
 For iframes, you can listen to network events and modify them, by using the `WebResourceRequested` event.
 
 See also:
-* [Manage network requests in WebView2](./overview-features-apis.md#manage-network-requests-in-webview2) in _Overview of WebView2 APIs_.
+* [Manage network requests in WebView2](./overview-features-apis.md#manage-network-requests-in-webview2) in _Overview of WebView2 APIs_.<!-- todo: delete the below tabset? -->
 * [Custom management of network requests](../how-to/webresourcerequested.md)
 * [Experimental APIs for 1.0.1222-prerelease](../release-notes/archive.md#experimental-apis-for-101222-prerelease) in _Archived Release Notes for the WebView2 SDK_.
 
 
+<!-- todo: delete tabset? -->
```

Comment:
delete tabset?  seems outdated and has notes saying to update in 2022

Reply:
won't update this h2 section in the present PR.  might enter work item after @sivMSFT investigates whether this feature is still Experimental.